test/ng/animateSpec.js

```diff
@@ -115,6 +115,18 @@ describe("$animate", function() {
       inject();
     });
 
+    it("should register the animation and be available for lookup", function() {
+      var provider;
+      module(function($animateProvider) {
+        provider = $animateProvider;
+      });
+      inject(function() {
+        expect(provider.$$registeredAnimations['filter']).toBeFalsy();
```

Comment:
check for hasOwnProperty instead, it's more generic and also add a comment that we are testing with a well known property name that is usually on objects or arrays, we must ensure that we don't create collisions with them